protocol/slp/slp.md

```diff
@@ -0,0 +1,78 @@
+# Simple Ledger Protocol
+
+Simple Ledger Protocol is a token system for the Bitcoin Cash network.
+It allows users to create, issue, and transfer digital tokens that enjoy the same security model and network of Bitcoin Cash.
+Users can associate the created tokens with assets and values, and thus, utilize the [Blockchain](/protocol/blockchain) as the public ledger to achieve transparency and integrity for their transactions.
+
+
+## The intent of token systems
+
+The Blockchain provides an open ledger with transparency and integrity to document transactions.
+However, to utilize this ledger, one has to devise rules and security models for the transactions, as well as the computational power to support a network that complies with the rules.
+The above costs make such applications prohibitively expensive for most individual users, while also hinders the emergence of coins for specific applications and transactions.
+Such needs drive the creation of various token systems on Bitcoin Cash.
+Token systems are additional layers of protocols on Bitcoin Cash, that allow the creation, issuance, and transfer of tokens on the Bitcoin Cash network.
+Users and creators of the tokens can utilize the computational power and other benefits of the Bitcoin Cash network for their own token transactions.
+
+
+## The design of Simple Ledger Protocol
+
+Simple Ledger Protocol (SLP) is one of the most prevalent token systems on Bitcoin Cash.
+SLP employs a “colored coins” design that associates token amounts with BCH [transaction](/protocol/blockchain/transaction) outputs.
+The first output of an SLP transaction will utilize the OP_RETURN to return messages with the predefined format to annotate the amount of SLP tokens associated with each transaction output in the same transaction.
+The message returned by OP_RETURN are from one of four predefined formats: [GENESIS](/protocol/slp/genesis), [MINT](protocol/slp/mint), [SEND](/protocol/slp/send), and [COMMIT](/protocol/slp/commit).
```

Comment:
Did you miss this or did you want to discuss it more.  Looking at it again in combination with the prior changes, I'm thinking we can actually just change it to something simpler, like: **The predefined messages formats include:**

Does that make sense?  My only real concerned here is the verb "return" -- I want to avoid making it sound like `OP_RETURN` is actually "returning" data.

Reply:
My bad, I missed this one last time. Indeed my language regarding OP_RETURN is rather misleading and I will change it to reflect the actual behavior.